### PR TITLE
[16.0][ADD] delivery_driver_stock_picking_batch

### PR DESCRIPTION
**Pull request by EmilioPascual**
Add drivers in stock picking batch

PR Series: https://github.com/OCA/delivery-carrier/pull/752 (must be merged before this one)

@Shide @gelojr @rafaelbn please review.

MT-4427 @moduon

### CODE REVIEW

File: delivery_driver_stock_picking_batch/models/stock_picking_batch.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+
+from odoo import api, fields, models
+
+
+class StockPickingBatch(models.Model):
+    _inherit = "stock.picking.batch"
+
+    driver_ids = fields.Many2many(
+        "res.partner", string="Drivers", compute="_compute_driver_ids", store=True
```
Comment (Shide) [inline delivery_driver_stock_picking_batch/models/stock_picking_batch.py]: Add the same domain as the stock.picking

```suggestion
        "res.partner", string="Drivers", compute="_compute_driver_ids", store=True, domain="[('is_company', '=', False)]",
```

**Review by Shide — CHANGES_REQUESTED** [review]
Good job!

**suggestion**: Add the same domain as the stock.picking

File: delivery_driver_stock_picking_batch/models/stock_picking_batch.py (hunk 1 above)
Comment (EmilioPascual) [reply delivery_driver_stock_picking_batch/models/stock_picking_batch.py]: Done. Thanks.

**Review by fcvalgar — APPROVED** [review]

**Comment by EmilioPascual** [comment]
https://www.loom.com/share/c53127a53b664a06b2e81c026a27cf88

**Review by fcvalgar — APPROVED** [review]
LGTM, thank you @EmilioPascual 

image

image

image


image

**Review by Shide — APPROVED** [review]

**Review by Gelojr — APPROVED** [review]
Nice job @EmilioPascual 
LGTM https://www.loom.com/share/a5520ca5fa7f4667be40aa535b40bbf2?sid=1187a7c8-a2a2-42e5-a08f-a252b0f64f7c

**Comment by EmilioPascual** [comment]
@rafaelbn can you review and merge if neccesary?

**Review by rafaelbn — APPROVED** [review]
Great. thank you! 👍🏼 

/ocabot merge patch

**Comment by EmilioPascual** [comment]
@rafaelbn I made rebase, please try again 🙏🏻

**Pull request merged**